change-validation/src/main/scala/com/normation/plugins/changevalidation/NotificationService.scala

```diff
@@ -128,13 +129,17 @@ class NotificationService(
   def sendNotification(step: WorkflowNode, cr: ChangeRequest): IOResult[Unit] = {
     for {
       serverConfig  <- getSMTPConf(configMailPath)
-      emailConf     <- getStepMailConf(step, configMailPath)
-      rudderBaseUrl <- getRudderBaseUrl(configMailPath)
-      params        =  extractChangeRequestInfo(rudderBaseUrl, cr)
-      mf            =  new DefaultMustacheFactory()
-      emailBody     <- getContentFromTemplate(mf, emailConf, params)
-      emailSubject  <- getSubjectFromTemplate(mf, emailConf.subject, params)
-      _             <- emailService.sendEmail(serverConfig, emailConf.toEnvelop(emailBody).copy(subject = emailSubject))
+      _             <- ZIO.when(!serverConfig.smtpHostServer.isBlank) {
```

Comment:
isEmpty not isBlank (we still support some old Java8 systems that does not support isBlank